features/step_definitions/cucumber_rails_steps.rb

```diff
@@ -15,12 +15,6 @@
   install_cucumber_rails :no_database_cleaner, :database_cleaner_active_record
 end
 
-Given('I have created a new Rails app and installed cucumber-rails without database_cleaner') do
-  rails_new
-  install_cucumber_rails :no_database_cleaner
-  overwrite_file('features/support/env.rb', "require 'cucumber/rails'\n")
-end
-
```

Comment:
I think this step is still used in `features/capybara_javascript_drivers.feature` and `features/database_cleaner.feature`.

Reply:
I have no idea why this is deleted. I didn't even delete it myself 👻